### PR TITLE
add cloud-ops-admin, add as admin for in all mojo and wtp accounts

### PR DESCRIPTION
**Pull request by bagg3rs**
Enabling AWS SSO for MoJ Official accounts 

Add cloud-ops-admin github group as admins on MoJO/WTP accounts
Add moj-official-techops as read-only on MoJO/WTP accounts
Add moj-official-techops as read-only on WTP accounts

### CODE REVIEW

File: terraform/sso-admin-account-assignments.tf [hunk 1]
```diff
@@ -140,12 +140,44 @@ locals {
...
-    # MOJ Official Technical Operations (Networking)
+    # MOJ Official Technical Operations (Networking) and Cloud Ops team
     {
       github_team    = "moj-official-techops"
       permission_set = aws_ssoadmin_permission_set.administrator-access
       accounts = [
-        aws_organizations_account.moj-official-network-operations-centre
+        aws_organizations_account.moj-official-network-operations-centre,
+        aws_organizations_account.wptpoc,
+        aws_organizations_account, workplace-tech-proof-of-concept-development
```
Comment (jakemulley) [inline terraform/sso-admin-account-assignments.tf:150]: Slight formatting issue here - should be `aws_organizations_account.workplace-tech-proof-of-concept-development`

File: terraform/sso-admin-account-assignments.tf [hunk 2]
```diff
@@ -140,12 +140,44 @@ locals {
...
+      accounts = [
+        aws_organizations_account.moj-official-network-operations-centre,
+        aws_organizations_account.moj-official-production,
+        aws_organizations_account.moj-official-pre-production,
+        aws_organizations_account.moj-official-development,
+        aws_organizations_account.moj-official-public-key-infrastructure-dev,
+        aws_organizations_account.moj-official-public-key-infrastructure,
+        aws_organizations_account.moj-official-shared-services,
+        aws_organizations_account.wptpoc,
+        aws_organizations_account, workplace-tech-proof-of-concept-development
```
Comment (jakemulley) [inline terraform/sso-admin-account-assignments.tf:165]: Slight formatting issue here

File: terraform/sso-admin-account-assignments.tf [hunk 3]
```diff
@@ -140,12 +140,44 @@ locals {
...
```
Comment (jakemulley) [inline terraform/sso-admin-account-assignments.tf:180]: Slight formatting issue here

**Review by jakemulley — CHANGES_REQUESTED** [review]

File: terraform/sso-admin-account-assignments.tf (hunk 1 above)
Comment (bagg3rs) [reply terraform/sso-admin-account-assignments.tf]: fixed

**Review by jakemulley — APPROVED** [review]

**Pull request merged**